source/blueprints/banking.rst

```diff
@@ -414,121 +341,76 @@ To see how the trained NLP pipeline performs on a test query, use the :meth:`pro
 
 .. code:: python
 
-   nlp.process("I'd like a mujaddara wrap and two chicken kebab from palmyra")
+   nlp.process('transfer two hundred dollars from my checking to my savings')
```

Comment:
Use ``""`` instead, since they allow for ``'`` to be incorporated in the text input.